README.md

```diff
@@ -72,11 +72,20 @@ santaRest.runAction(new ExampleAction());
 
 To receive actions with responses, you should subscribe to events:
 ```java
-santaRest.subscribe(this);
+santaRest.getActionPoster().subscribe(this);
 ```
```

Comment:
Review code please.
There are 2 handler variants:
1. ActionPoster
2. rx.Observable.
I think, we need to split this variants to do that more writeable. And for user `subscribe` isn't clear for understanding. In my variant we have:
santaRest.getActionPoster().subscribe(..)
santaRest.observeActions().subscribe(..)